src/main/java/io/jenkins/plugins/orka/helpers/FormValidator.java

```diff
@@ -51,6 +55,28 @@ public FormValidation doCheckConfigName(String configName, String orkaEndpoint,
         return FormValidation.ok();
     }
 
+    public FormValidation doCheckImage(String image) {
+        Jenkins.get().checkPermission(Jenkins.ADMINISTER);
+
+        String invalidImageError = "Invalid image name. Image must be an OCI reference or present on Orka SAN Storage.";
```

Comment:
`nit` - we can move this outside of the method.
And we should have tests for this